app/assets/stylesheets/sign_in.scss

```diff
@@ -0,0 +1,12 @@
+@import "colors";
+
+.sign-up {
+  background-image: url("book2.jpg");
+  border-radius: 20px;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size, border-radius, padding-bottom, padding-top, text-align